veascan-web/src/Navbar/SearchBar.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import SearchIcon from "tsx:svgs/icons/search.svg";
+import { smallScreenStyle } from "styles/smallScreenStyle";
+
+const Container = styled.div`
+  height: 45px;
+  margin-top: 9px;
+  margin-bottom: 10px;
+  width: 540px;
+  display: flex;
+  background-color: transparent;
+  color: #becce5;
+  border: 1px solid #42498f;
+  border-radius: 3px;
+
+  ${smallScreenStyle(css`
+    height: calc(32px + (45 - 32) * (100vw - 300px) / (1250 - 300));
+  `)}
+
+  ::placeholder {
```

Comment:
try to not hardcode colors, let me know if you need this color added to the theme.